src/main/java/net/gigaclub/builderteamsplugin/Commands/Worlds.java

```diff
@@ -0,0 +1,244 @@
+
+                                }
+                            }
+                        }
+                    } else if (args.length >= 4) {
+                        for (Object o : builderSystem.getAllWorldTypes()) {
+                            HashMap m = (HashMap) o;
+                            String worldTyp = (String) m.get("name");
+                            String sworldTyp = stackworldtyp(args, 4);
+                            if (sworldTyp.equalsIgnoreCase(worldTyp.toLowerCase())) {
+                                if (isInt(args[1])) {
+                                    builderSystem.createWorldAsUser(playerUUID, Integer.parseInt(args[1]), args[2], worldTyp);
+                                }
+                            }
+                        }
+                        break;
+                    } else if (args.length >= 6) {
+                        player.sendMessage(t.t("BuilderSystem.toomany_Arguments", playerUUID));
+                        return false;
+                    }
+                case "remove":
+
+                    System.out.println("GetTeam ? " + builderSystem.getTeam(args[1]));
+                    if (!(ownteamname.equalsIgnoreCase(args[1]))) {
+                        String Teamname = (String) builderSystem.getTeam(args[1].toLowerCase());
+                        if (isInt(args[2])) {
+
+                            builderSystem.removeTeamFromWorld(playerUUID, Teamname, Integer.parseInt(args[2]));
+                        }
+                    }
+                    break;
+                case "addteam":
+
+                    if (!(ownteamname.equalsIgnoreCase(args[1]))) {
+                        String Teamname = (String) builderSystem.getTeam(args[1]);
+                        if (isInt(args[2])) {
+
+                            builderSystem.addTeamToWorld(playerUUID, Teamname, Integer.parseInt(args[2]));
+                        }
+                    }
+                    break;
+                case "adduser":
+                    if (isInt(args[2])) {
+                        String addetuser = Bukkit.getPlayer(args[1]).toString();
+
+                        builderSystem.addTeamToWorld(playerUUID, addetuser, Integer.parseInt(args[2]));
+
+                    }
+                    break;
+                case "list":
+                    for (Object o : builderSystem.getAllWorlds()) {
+                        HashMap m = (HashMap) o;
+                        player.sendMessage(ChatColor.GRAY + "ID: " + ChatColor.WHITE + m.get("world_id").toString());
+                        player.sendMessage(ChatColor.GRAY + "Name: " + ChatColor.WHITE + m.get("name").toString());
+                        player.sendMessage(ChatColor.GRAY + "World typ:" + ChatColor.WHITE + m.get("world_type").toString());
+                     player.sendMessage("Teams");
+                        System.out.println((String) m.get("team_ids"));
```

Comment:
@Feier68 

`for (Object team : (Object[]) m.get("team_ids") {
      HashMap teamMap = (HashMap) team;
      ........
}`

Reply:
`for (HashMap h : (HashMap[]) m.get("team_ids")) {`

Reply:
```
[13.08 20:47:30.144] INFO: [Lobby-1] [20:47:03 ERROR]: null
[13.08 20:47:30.144] INFO: [Lobby-1] org.bukkit.command.CommandException: Unhandled exception executing command 'gcprojekt' in plugin Builder_Teams_Plugin v1.17.1.0.0
[13.08 20:47:30.144] INFO: [Lobby-1]    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2185) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:1996) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:1977) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:6) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:36) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1340) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.shouldRun(MinecraftServer.java:193) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1273) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1184) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    at java.lang.Thread.run(Thread.java:831) [?:?]
[13.08 20:47:30.144] INFO: [Lobby-1] Caused by: java.lang.ClassCastException: class [Ljava.lang.Object; cannot be cast to class java.lang.String ([Ljava.lang.Object; and java.lang.String are in module java.base of loader 'bootstrap')
[13.08 20:47:30.144] INFO: [Lobby-1]    at net.gigaclub.builderteamsplugin.Commands.Worlds.onCommand(Worlds.java:136) ~[?:?]
[13.08 20:47:30.144] INFO: [Lobby-1]    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper.jar:git-Paper-53]
[13.08 20:47:30.144] INFO: [Lobby-1]    ... 21 more
```

Reply:
[13.08 20:50:10.467] INFO: [Lobby-1] [20:49:52 ERROR]: null
[13.08 20:50:10.467] INFO: [Lobby-1] org.bukkit.command.CommandException: Unhandled exception executing command 'gcprojekt' in plugin Builder_Teams_Plugin v1.17.1.0.0
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2185) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:1996) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:1977) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:6) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:36) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1340) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.shouldRun(MinecraftServer.java:193) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1273) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1184) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at java.lang.Thread.run(Thread.java:831) [?:?]
[13.08 20:50:10.467] INFO: [Lobby-1] Caused by: java.lang.ClassCastException: class [Ljava.lang.Object; cannot be cast to class [Ljava.util.HashMap; ([Ljava.lang.Object; and [Ljava.util.HashMap; are in module java.base of loader 'bootstrap')
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.gigaclub.builderteamsplugin.Commands.Worlds.onCommand(Worlds.java:137) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    ... 21 more

Reply:
```
[13.08 20:50:10.467] INFO: [Lobby-1] [20:49:52 ERROR]: null
[13.08 20:50:10.467] INFO: [Lobby-1] org.bukkit.command.CommandException: Unhandled exception executing command 'gcprojekt' in plugin Builder_Teams_Plugin v1.17.1.0.0
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:821) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2185) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:1996) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:1977) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:6) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:36) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1340) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.shouldRun(MinecraftServer.java:193) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1273) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1184) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    at java.lang.Thread.run(Thread.java:831) [?:?]
[13.08 20:50:10.467] INFO: [Lobby-1] Caused by: java.lang.ClassCastException: class [Ljava.lang.Object; cannot be cast to class [Ljava.util.HashMap; ([Ljava.lang.Object; and [Ljava.util.HashMap; are in module java.base of loader 'bootstrap')
[13.08 20:50:10.467] INFO: [Lobby-1]    at net.gigaclub.builderteamsplugin.Commands.Worlds.onCommand(Worlds.java:137) ~[?:?]
[13.08 20:50:10.467] INFO: [Lobby-1]    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper.jar:git-Paper-53]
[13.08 20:50:10.467] INFO: [Lobby-1]    ... 21 more
```

Reply:
```
                       for (HashMap team : (HashMap[]) m.get("team_ids")) {
                           HashMap teamMap = (HashMap) team;
                            Player player1 = Bukkit.getPlayer(team.get("name").toString());
                            String player2 = player1.toString();
                            StringBuilder res = new StringBuilder();
                            res.append(player2).append(ChatColor.WHITE + " , " + ChatColor.GRAY);
                            String strValue = "ChatColor.GRAY +";
                            res.append(new StringBuilder(strValue).reverse());
                            res.toString();

                            player.sendMessage(res.toString());
                            }
                        player.sendMessage("");
                       player.sendMessage("User: ");

                        for (HashMap user : (HashMap[]) m.get("user_ids")) {
                            HashMap userMap = (HashMap) user;

                            Player player11 = Bukkit.getPlayer(user.get("name").toString());
                            String player21 = player11.toString();
                            StringBuilder res1 = new StringBuilder();
                            res1.append(player21).append(ChatColor.WHITE + " , " + ChatColor.GRAY);
                            String strValue = "ChatColor.GRAY +";
                            res1.append(new StringBuilder(strValue).reverse());
                            res1.toString();

                            player.sendMessage(res1.toString());
                        }
```

Reply:
for (Object team : (Object[]) m.get("team_ids")) {
                           HashMap teamMap = (HashMap) team;
                            Player player1 = Bukkit.getPlayer(team.get("name").toString());
                            String player2 = player1.toString();
                            StringBuilder res = new StringBuilder();
                            res.append(player2).append(ChatColor.WHITE + " , " + ChatColor.GRAY);
                            String strValue = "ChatColor.GRAY +";
                            res.append(new StringBuilder(strValue).reverse());
                            res.toString();

                            player.sendMessage(res.toString());
                            }
                        player.sendMessage("");
                       player.sendMessage("User: ");

                        for (Objectuser : (Object[]) m.get("user_ids")) {
                            HashMap userMap = (HashMap) user;

                            Player player11 = Bukkit.getPlayer(user.get("name").toString());
                            String player21 = player11.toString();
                            StringBuilder res1 = new StringBuilder();
                            res1.append(player21).append(ChatColor.WHITE + " , " + ChatColor.GRAY);
                            String strValue = "ChatColor.GRAY +";
                            res1.append(new StringBuilder(strValue).reverse());
                            res1.toString();

                            player.sendMessage(res1.toString());
                        }

Reply:
- world_id
- name
- task_id
- world_type
- team_manager_ids
- team_ids
- user_manager_ids
- user_ids

Reply:
![Unbenannt](https://user-images.githubusercontent.com/25251081/129407417-adb01077-8acd-4b85-8894-854ee3888c42.PNG)